src/main/java/edu/harvard/iq/dataverse/trsa/registry/AbstractFacade.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.harvard.iq.dataverse.trsa.registry;
+
+import java.util.List;
+import javax.persistence.EntityManager;
+
+/**
+ *
+ * @author asone
+ */
+public abstract class AbstractFacade<T> {
```

Comment:
`AbstractFacade` seems like too generic of a name. Should it be `TrsaAbstractFacade`?